samples/CommunityToolkit.Maui.Sample/ViewModels/Converters/ByteArrayToImageSourceConverterViewModel.cs

```diff
@@ -50,15 +50,15 @@ async Task DownloadDotNetBotImage(CancellationToken token)
 
 		try
 		{
-			DotNetBotImageByteArray = await client.GetByteArrayAsync("https://user-images.githubusercontent.com/13558917/137551073-ac8958bf-83e3-4ae3-8623-4db6dce49d02.png", maximumDownloadTimeCTS.Token).WaitAsync(token).ConfigureAwait(false);
+			DotNetBotImageByteArray = await client.GetByteArrayAsync($"https://user-images.githubusercontent.com/13558917/137551073-ac8958bf-83e3-4ae3-8623-4db6dce49d02.png", maximumDownloadTimeCTS.Token).WaitAsync(token).ConfigureAwait(false);
 
 			await minimumDownloadTimeTask.ConfigureAwait(false);
 
 			LabelText = "The above image was downloaded as a byte[]";
 		}
 		catch (Exception e)
 		{
-			Trace.WriteLine(e);
+			Trace.TraceInformation(e.Message);
```

Comment:
This will lose the stack trace information from the exception. Is this something that we want?

Reply:
Agreed, it's always best to print the entire Exception 

```suggestion
			Trace.TraceInformation(e);
```

Reply:
Yes I think I should change that. Good catch.